discovery_engine_core/web-api/src/db.rs

```diff
@@ -12,82 +12,48 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-use serde_json::from_reader;
-use std::{collections::HashMap, fs::File, path::PathBuf, sync::Arc};
+use std::sync::Arc;
 use xayn_discovery_engine_ai::{CoiConfig, CoiSystem, GenericError};
-use xayn_discovery_engine_bert::{AveragePooler, SMBert, SMBertConfig};
-use xayn_discovery_engine_tokenizer::{AccentChars, CaseChars};
 
-use crate::{
-    elastic,
-    models::{IngestedDocument, PersonalizedDocument},
-    storage::UserState,
-};
+use crate::{elastic, elastic::ElasticState, storage::UserState};
 
 pub(crate) type Db = Arc<AppState>;
```

Comment:
❓ the type alias is a bit misleading/not up to date anymore, eg its coi system has nothing to do with a db. maybe we should just remove the alias all together? writing the whole type is not a big deal.

❓ at the same time, maybe also the module should be renamed from `db` to `state`?